libvast/vast/chunk.hpp

```diff
@@ -81,8 +81,12 @@ class chunk final : public caf::ref_counted {
     requires(!(std::is_lvalue_reference_v<
                  Buffer> || std::is_trivially_move_assignable_v<Buffer>))
   static auto make(Buffer&& buffer) -> decltype(as_bytes(buffer), chunk_ptr{}) {
```

Comment:
Btw, is there a special reason why we kept the `as_bytes(buffer)` as SFINAE and didn't move it into the `requires` clause?

Reply:
I've pushed the change.